python/setup.py

```diff
@@ -81,12 +81,15 @@ def has_ext_modules(self):
       # The BinaryDistribution argument triggers build_ext.
       distclass=BinaryDistribution,
       install_requires=["numpy",
+                        "cython",
```

Comment:
If you build your wheel correctly, cython should only be a build/setup_requires but not needed for installation.

Reply:
Thanks! Trying to fix this in #878.